### ISSUE TITLE
On Boyue Likebook Alita, in KO Reader, one finger top edge downswipe to bring up system menu causes hard exit.

### ISSUE THREAD

**Issue by squiresque** [koreader/koreader#7651]
* KOReader version: 2021.04-47
* Device: Likebook Boyue Alita, Android 8.1, BYRead-v3 1.0-2021050714

#### Issue
I have been updating regularly, usually daily, with the newest releases and in the last couple of weeks KO Reader has been doing a hard exit without saving any history information after doing a top edge down one finger swipe to bring up the Alita system menu.  I usually do this to switch to the file manager to install a KO Reader update apk file.  Previously the system menu would appear on the top, I would hit the background app icon, then select the file manager, and then install the new KO Reader APK file.  After installing the KO Reader update, I would open KO Reader and would be on the page I was reading prior to the update.   Now, after bringing up the system menu on the top edge, KO Reader exits, and when I return to KO Reader after updating, there is no memory of my place in a book, or in some cases of the book that was opened prior to the exit.  I tried different KO Reader persistence settings - both update every five minutes, and update after save or exit - and neither help.  

#### Steps to reproduce
On the Alita, top edge one finger down swipe to bring up system menu.  Evidence that some kind of ungraceful exit occurred is that KO Reader loses memory of the current place in a book, or in the book that was open prior to the exit. 

##### `crash.log` (if applicable)
`crash.log` is a file that is automatically created when KOReader crashes. It can
normally be found in the KOReader directory:

* `/mnt/private/koreader` for Cervantes
* `koreader/` directory for Kindle
* `.adds/koreader/` directory for Kobo
* `applications/koreader/` directory for Pocketbook

Android won't have a crash.log file because Google restricts what apps can log, so you'll need to obtain logs using `adb logcat KOReader:I ActivityManager:* AndroidRuntime:* DEBUG:* *:F`.

Please try to include the relevant sections in your issue description.
You can upload the whole `crash.log` file on GitHub by dragging and
dropping it onto this textbox.

If you instead opt to inline it, please do so behind a spoiler tag:
<details>
  <summary>crash.log</summary>
  
```
<Paste crash.log content here>
```
</details>

**Comment by pazos** [comment]
We cannot guess what happens without logs.

**Comment by squiresque** [comment]
Well, before I was able to figure out how to use a command line invocation of KOReader on my Alita to generate a crash log   -- something I have yet to do -- the issue was fixed by a more recent update.  Moreover, no doubt moved to pity by my description of my method of updating KOReader, KOReader now allows me to install the updates without leaving KOReader at all, which is awesome.  Thank you so very much!
